client/src/components/ButtonSubmit/ButtonSubmit.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap');
+
+.ButtonSubmit {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  font-family: 'Reem Kufi', sans-serif;
+}
+
+.ButtonSubmit-followed,
+.ButtonSubmit-favorited {
+  background: #FB836F;
+  border-radius: 4px;
+  width: 200px;
+  height: 50px;
+  color: #FAFAFA;
+  font-size: 18px;
+}
```

Comment:
Rather than giving these an explicit height and width, we want these to have padding ratio that is consistent across all the button types.

We may also want to give these a `max-width` property incase the text is really long for some reason